fft_simulation/fft_simulation.py

```diff
@@ -102,31 +102,19 @@ def save_to_nifti(data, image_resolution, output_path):
 
 
 
-@click.command(help="Compute the magnetic field Bz in ppm from a susceptibility distribution in NIfTI format.")
+@click.command(help="Compute the magnetic field variation in ppm from a susceptibility distribution in NIfTI format.")
```

Comment:
Use @click.option rather than @click.argument when possible. Options are more clear and it's been the standard in ST.

Reply:
Got it. Will change to @click.option.